cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -63,6 +63,17 @@ public class KafkaBrokerConfigurationBuilder {
     private final StringWriter stringWriter = new StringWriter();
     private final PrintWriter writer = new PrintWriter(stringWriter);
     private final Reconciliation reconciliation;
+
+    /**
+     * The configuration field name for Kafka metric reporters.
+     */
+    public static final String KAFKA_METRIC_REPORTERS_CONFIG_FIELD = "metric.reporters";
+
+    /**
+     * The class name for the Strimzi Metrics Reporter.
+     */
+    public static final String STRIMZI_METRIC_REPORTER = "io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter";
```

Comment:
This is a duplication of the constant in `StrimziReporterMetricsModel`.

Reply:
Removed